api/config/sample_data/config_request.go

```diff
@@ -0,0 +1,60 @@
+package sample_data
+
+import (
+	config "github.com/chef/automate/api/config/shared"
+	w "github.com/chef/automate/api/config/shared/wrappers"
+)
+
+// NewConfigRequest returns a new instance of ConfigRequest with zero values.
+func NewConfigRequest() *ConfigRequest {
+	return &ConfigRequest{
+		V1: &ConfigRequest_V1{
+			Sys: &ConfigRequest_V1_System{
+				Mlsa:    &config.Mlsa{},
+				Tls:     &config.TLSCredentials{},
+				Service: &ConfigRequest_V1_System_Service{},
+				Log:     &ConfigRequest_V1_System_Log{},
+			},
+			Svc: &ConfigRequest_V1_Service{},
+		},
+	}
+}
+
+// DefaultConfigRequest returns a new instance of ConfigRequest with default values.
+func DefaultConfigRequest() *ConfigRequest {
+	c := NewConfigRequest()
+	c.V1.Sys.Service.Host = w.String("127.0.0.1")
+	c.V1.Sys.Service.Port = w.Int32(10155)
```

Comment:
How did you determine this value?

Reply:
We can pick any of the port which are available in `chef-automate dev ports free`

Reply:
Cool; did not know that command!